### DISCUSSION TITLE
Modal footer in modal component dont hide the buttons

### DISCUSSION THREAD

**Discussion by lucapu88** [bootstrap-vue/bootstrap-vue#6228] [Q&A] [answered]
Sorry if I open a thousand reports here on github, unfortunately I'm not on the discord channel, I still have a lot to open... : P

![Schermata da 2020-12-18 15-33-46](https://user-images.githubusercontent.com/56302007/102626111-afed1b00-4146-11eb-9c6d-259a3b9821f0.png)

```
    <b-modal scrollable id="modal-2" modal-footer>
       <p>bla bla bla</p>
    </b-modal>
```

I did as it says in the picture, but I still see the cancel and ok buttons ... how can I delete them ???

**Comment by Hiws** [comment, marked as answer]
You're not using the `modal-footer` **slot**, you've added a `modal-footer` **prop**.

The markup should be.

```html
<b-modal>
  <template #modal-footer>
    <p>bla bla bla</p>
  </template>
</b-modal>
```

You can take a look at the Vue docs if want more information about how slots work.
https://vuejs.org/v2/guide/components-slots.html#Named-Slots

**Reply by Hiws** [reply]
I should probably note that if you just want to hide the footer completely, and not place any content in it.
You can use the `hide-footer` **prop**, instead of using the slot.

**Comment by Hiws** [comment]
For future Questions, please use the `Discussion` section on our GitHub page.
You can then create it under the `Q&A` category.
<https://github.com/bootstrap-vue/bootstrap-vue/discussions>
